### DISCUSSION TITLE
Impossible to cancel orders created for Seaport 1.5 after migration to opensea-js 7.1.1 (Seaport 1.6)

### DISCUSSION THREAD

**Discussion by aashborn** [ProjectOpenSea/opensea-js#1428] [General]
After upgrading to opensea-js 7.1.1, I find that I cannot cancel orders created by opensea-js before the upgrade:

```
Only 0x0000000000000068F116a894984e2DB1123eB395 is currently supported for cancelling orders.
    at OpenSeaSDK.cancelSeaportOrders (opensea-js.js?v=a731faac:62670:17)
    at OpenSeaSDK.cancelOrder (opensea-js.js?v=a731faac:62689:44)
```

This is how I cancel a listing (pseudo-code)
```
                const response = await openseaSDK.api.getOrders(...);
                const order = response.orders.find(...)
                await this.openseaSDK.cancelOrder({
                order
            });
```

With this error, I updated the order's `protocolAddress `field with Seaport 1.6:

`order.protocolAddress = "0x0000000000000068F116a894984e2DB1123eB395";
`

The error no longer occurs, but the transaction does nothing, the token is still listed. 

Any idea how to solve this problem?

Thanks

**Comment by ryanio** [comment]
This was fixed in v7.1.2, can you give that a try https://github.com/ProjectOpenSea/opensea-js/releases/tag/v7.1.2

**Reply by aashborn** [reply]
The fix works. Thank you!
